test/integration/platform_admin_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
```

Comment:
This was generated using rails g integration_test.  How about them apples hound?